lib/chef/knife/vsphere_vm_wait_sysprep.rb

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
You should run rubocop on this I know the rest of the project has changed the options to using the var: method instead of using :var =>